### ISSUE TITLE
Cannot restore Mac to Linux (hfsplus drive)

### ISSUE THREAD

**Issue by anarchotaoist** [borgbase/vorta#753]
**Describe the bug**

Cannot restore Mac vorta backup to Linux desktop with Vorta

**To Reproduce**
A Vorta backup is made to an external Lacie hfsplus formatted drive successfully from a Macbook pro.
The drive is then plugged into a Linux machine.
Open Vorta, click repository,
add existing repository
Get message: Validating exisisting repo
Get message: Unable to add your repository

**Desktop (please complete the following information):**
- OS: Macos High Sierra 10.13.6
- Vorta version: 0.6.20 (nb the newest version would not install on 10.13)

- OS:Operating System: Kubuntu 20.10
KDE Plasma Version: 5.19.5
KDE Frameworks Version: 5.74.0
Qt Version: 5.14.2
Kernel Version: 5.8.0-36-generic
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-6700 CPU @ 3.40GHz
Memory: 31.2 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 530
- Vorta version: 0.7.1

**Additional context**
I tested first on a USB stick formatted with fat32 and it worked! 
It does not work on the hfsplus drive. Could this be the issue?
I also tried a backup without encryption but this did not work work either.
[vorta.log](https://github.com/borgbase/vorta/files/5790017/vorta.log)

log attatched

**Comment by m3nu** [comment]
Looks like a permission issue.

`Read-only file system: '/media/anarcho/LaCie/vortborg/lock.exclusive`

**Comment by anarchotaoist** [comment]
Thanks for the reply.
So, this is to say the drive itself is read only?
nb. It has existing TimeMachine backups. I was wanting to test Vorta first, and then replace the TimeMachine backups with a full Vorta backup of the mac files.
If I were to re-format it, with read/write permission then it should work - on hfsplus - or is another file system better?
Thanks.

**Comment by samuel-w** [comment]
To write hfsplus on Linux, you have to have uid 501 https://help.ubuntu.com/community/hfsplus.

**Comment by anarchotaoist** [comment]
Ok, thanks for your help.
Firstly, I changed permissions. This did not help.

I then used the diskutil in Macos to disable journaling for the Vorta backups. The terminal said it was already disabled.
I did not disable journaling for the whole drive.

re: the section Enable HFS+ read/write for a non-root Ubuntu user
I am a bit concerned about "changing your Ubuntu User ID (UID)"

The NFS file format seems to much of a headache to deal with to be honest.

As MacOS reads exFAT, could I simply format the drive as exFAT to use Vorta across the two OS's without issue? 
Does file size matter? 900GB
Thanks.
